hack/header_file

```diff
@@ -0,0 +1,16 @@
+/*
```

Comment:
Can't we sync it with ./hack/boilerplate.txt and use same header here and in controller-gen?

Reply:
@k0da thanks, i've made boilerplate a single source of headers file